exercises/src/components/exercise/tags.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { Row } from 'react-bootstrap';
+import { Alert } from'react-bootstrap';
```

Comment:
whitespace

Reply:
hah, and also duplicated import, could have been:
`import { Row, Alert } from 'react-bootstrap';`